packages/core-components/src/components/Select/Select.tsx

```diff
@@ -223,8 +246,14 @@ export function SelectComponent(props: SelectProps) {
                     key={items.find(el => el.value === selectedValue)?.value}
                     label={items.find(el => el.value === selectedValue)?.label}
                     clickable
-                    onDelete={handleDelete(selectedValue)}
+                    deleteIcon={
+                      <Typography
+                        className={classes.deleteButton}
+                        onMouseDown={event => event.stopPropagation()}
+                      />
+                    }
                     className={classes.chip}
+                    onDelete={e => handleDelete(e, selectedValue)}
```

Comment:
This would not be ideal. `"@material-ui/icons"` exist as a dependency, which contains the `DeleteIcon`.

Reply:
I tried but i was not able to find the `@material-ui/icons`.